.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -18,9 +18,7 @@ assignees: ''
 
       git checkout -b pr/prepare-$RELEASE
       ./contrib/update-helm-chart.sh $RELEASE
-      pushd install/kubernetes/tetragon
-      ./test.sh
-      popd
+      ./install/kubernetes/test.sh
```

Comment:
Indeed this is no longer needed, you saw this https://github.com/cilium/tetragon/commit/357206b421615ad16a46da821582ac0dd00d21d0, you have 👀  everywhere.